doc/man7/provider-digest.pod

```diff
@@ -92,7 +94,8 @@ macros in L<openssl-core_dispatch.h(7)>, as follows:
 
 A digest algorithm implementation may not implement all of these functions.
 In order to be usable all or none of OSSL_FUNC_digest_newctx, OSSL_FUNC_digest_freectx,
-OSSL_FUNC_digest_init, OSSL_FUNC_digest_update and OSSL_FUNC_digest_final should be implemented.
+OSSL_FUNC_digest_copyctx, OSSL_FUNC_digest_init, OSSL_FUNC_digest_update
+and OSSL_FUNC_digest_final should be implemented.
```

Comment:
Not sure I get. It looks like OSSL_FUNC_digest_copyctx is not mandatory to implement with other 4...

Reply:
> Not sure I get. It looks like OSSL_FUNC_digest_copyctx is not mandatory to implement with other 4...

Thanks for your review, you are right, OSSL_FUNC_digest_copyctx is not necessary to implement.